cmake/ctest/drivers/atdm/chama/drivers/Trilinos-atdm-chama-intel-debug-openmp-panzer.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 export SALLOC_CTEST_TIME_LIMIT_MINUTES=0:45:00
 if [ "${Trilinos_TRACK}" == "" ] ; then
-  export Trilinos_TRACK=ATDM
+  export Trilinos_TRACK=Selected
```

Comment:
Should these be:
 `export Trilinos_TRACK=Specialized` 
instead of 
`export Trilinos_TRACK=Selected`